Detectors/MUON/MCH/Workflow/src/TBDigitsFileReaderSpec.cxx

```diff
@@ -0,0 +1,91 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TBDigitsFileReaderSpec.cxx
+/// \brief Implementation of a data processor to run the preclusterizer
+///
+/// \author Philippe Pillot, Subatech
```

Comment:
I guess you copy/past these comments ;-)

Reply:
Yes, I corrected it.